include/common/Driver.hpp

```diff
@@ -253,23 +253,23 @@ class Driver
 
 		// Configure main solver parameters
 		pp.pushScope("solver");
-		_sim->configure(pp);
-		
+
 		// Configure section times
 		std::vector<double> secTimes;
 		std::vector<bool> secCont;
 		extractSectionTimes(pp, secTimes, secCont);
+		_sim->setSectionTimes(secTimes, secCont);
+		_sim->configure(pp);
 
 		pp.popScope(); // solver scope
 
+		// Create and configure model
 		pp.pushScope("model");
 
-		// Create and configure model
 		cadet::IModelSystem* model = _builder->createSystem(pp);
 
 		// Hand model over to simulator
 		_sim->initializeModel(*model);
-		_sim->setSectionTimes(secTimes, secCont);
```

Comment:
If this is deleted, section times will not be propagated into the models. But, this is required for the inlet profiles to work.